controllers/configmap_controller.go

```diff
@@ -67,9 +67,22 @@ const (
 	ConfigMapController = "configmap"
 )
 
+// serviceConfig defines the contents of the Windows service ConfigMap
+type serviceConfig struct {
```

Comment:
Would the [`servicescm.data`](https://github.com/openshift/windows-machine-config-operator/blob/master/pkg/servicescm/servicescm.go#L91) struct be useful here? I feel like exporting it would allow you to remove this `serviceConfig` introduction.

newServiceConfig() would stay our source of truth for services CM data then return `*servicescm.Data`.

Reply:
I think it would be, i'll make that change